src/twisted/conch/newsfragments/10138.bugfix

```diff
@@ -0,0 +1 @@
+twisted.conch.ssh.Factory.buildProtocol now sets transport's supportedPublicKeys to a list, like it was in Python 2.x. This prevents an exception when we try to subscript that list in SSHServerTransport.ssh_KEXINIT().
```

Comment:
This message should mention Dropbear compatibility as that's the user-visible symptom of the issue.